universe/info.owlbot/device.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
```

Comment:
It is customary at the top of source files to include a copyright comment with the date and authorship. It's your contribution so you own copyright on it, and you'll want people to know!